src/util/config-file.js

```diff
@@ -43,7 +44,7 @@ function writeConfig(configPath, content) {
       `Error! Project files was overridden and is empty in ${configPath}`,
     )
   }
-  return pify(fs.writeFile)(configPath, `${JSON.stringify(content, null, 2)}\n`)
+  return pify(fs.writeFile)(configPath, `${formatConfig(content)}\n`)
```

Comment:
It seems that we pass the content as config, no?

Reply:
Here we are replacing this code by `JSON.stringify(undefined, null, 2)`

Reply:
🌷🌷